source/guide_gitea.rst

```diff
@@ -29,62 +28,124 @@ of the then-popular Gogs_, but quickly became more active and bigger than the or
   * :manual:`supervisord <daemons-supervisord>`
   * :manual:`domains <web-domains>`
 
+
 Prerequisites
 =============
 
-.. include:: includes/my-print-defaults.rst
```

Comment:
This snippet is included in all guides, which use the mysql server so readers know where to get the mysql PW. Please use it instead of the more pipey-version below. If you'd like to change this, please open an issue so we can change the snippet and improve all mysql dependent guides that way.

Reply:
Done